### PR TITLE
Add lvis via grayskull

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-python @ngam this is ready for review. Thanks!

File: recipes/lvis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lvis-{{ version }}.tar.gz
+    sha256: 55aeeb84174abea2ed0d6985a8e93aa9bdbb60c61c6db130c8269a275ef61a6e
+  - path: requirements.txt
```
Comment (beenje) [inline recipes/lvis/meta.yaml:11]: Looks like the project isn't really maintained but could you maybe open an issue at https://github.com/lvis-dataset/lvis-api/issues to mention that both `LICENSE` and `requirements.txt` are missing from the sdist?

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/lvis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+test:
+  imports:
+    - lvis
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/lvis/
```
[the remark this answers is not in the document]
Comment (Tobias-Fischer) [reply recipes/lvis/meta.yaml]: Done.

File: recipes/lvis/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipes/lvis/meta.yaml:11]: Done: https://github.com/lvis-dataset/lvis-api/issues/33

**Pull request merged**